Makefile.toml

```diff
@@ -26,13 +26,12 @@ ICU4X_BUILDING_WITH_FORCED_NIGHTLY = { value = "1", condition = { env_set = ["IC
 [tasks.quick]
 description = "Run quick version of all lints and builds (useful before pushing to GitHub)"
 category = "ICU4X Development"
-dependencies = [
+env = { "ICU4X_DATA_DIR" = "${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/provider/datagen/tests/data/baked" }
+run_task.name = [
     "ci-job-fmt",
     "ci-job-clippy",
-    "ci-job-nostd",
```

Comment:
nit: reorder the jobs in `ci-job-all` as well

Reply:
I did not change the order; I removed some things from the job entirely. I figured it was fairly rare for the nostd job to randomly fail, so I just removed it from `cargo make quick`. Same with `tutorials-local` and certain parts of `tidy`. Everything still runs on CI; this is just to catch the things that commonly don't work, like Clippy, Fmt, and Docs.

Reply:
Yes, and the first section in `ci-job-all` that says `# make quick` still contains these. You should move them after that section if you have determined that they are too slow to run in `quick`.

Reply:
#4559